docs/organization/membership/index.mdx

```diff
@@ -35,7 +35,7 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can view and act on issues (such as assigning and resolving)                    |             | ✓          | ✓         | ✓           | ✓         |
 | Can join and leave existing teams                                               |             | ✓          | ✓         | ✓           | ✓         |
 | Can add repositories (through GitHub integrations)                              |             | ✓          | ✓         | ✓           | ✓         |
-| Can create and remove teams from the organization                               |             |            | ✓         | ✓           | ✓         |
+| Can create and remove teams from the organization                               |             | ✓\*\*      | ✓         | ✓           | ✓         |
```

Comment:
As this comes before "Can create and remove projects for the organization", should we swap the footnotes? Here will be `✓\*` and `✓\*\*` is below.

Reply:
Good suggestion! I've fixed the ordering